### PR TITLE
feat: add footer

### LINKED ISSUE #36: Add a footer
It could be nice to have a footer with a simple message like "Powered by [Dashlog](https://github.com/dashlog/web-ui/i)" with a link to GitHub.

This addition will effectively promote awareness of the tool, especially when it's deployed.

### PR DESCRIPTION
**Pull request by The-Alan-Turing**
Fixes #36

Adding a footer with a simple message like "Powered by [Dashlog](https://github.com/dashlog/web-ui/)" with a link to GitHub.

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -44,4 +44,5 @@
   <!-- EJS generated server side -->
   <header></header>
   <main id="data"></main>
+  <footer></footer>
```
Comment (PierreDemailly) [inline public/index.html:47]:
```suggestion
  <footer>
  	<span class="footer-text">
  	  Powered by <b><a target="_blank" href="https://github.com/dashlog/web-ui">Dashlog</a></b>
  	</span>
  </footer>
```
I also wonder if `<strong>` wouldn't be better semantically than `<b>`.

> The `<strong>` element is for content that is of greater importance, while the `<b>` element is used to draw attention to text without indicating that it's more important.

**Review by PierreDemailly — CHANGES_REQUESTED** [review]
Hi @The-Alan-Turing, thanks for contributing :)

Since the footer doesn't change depending on the org statusboard, it's not necessary to create a dynamic ejs view.

You can simply add the footer in the `index.html` as suggested

File: public/index.html (hunk 1 above)
Comment (The-Alan-Turing) [reply public/index.html]: Yes, I agree with using `<strong>` tag here. 
I have updated the PR.

**Review by PierreDemailly — APPROVED** [review]

**Comment by PierreDemailly** [comment]
@allcontributors please add @The-Alan-Turing for code

**Pull request merged**